src/test/histogram_test.c

```diff
@@ -71,6 +71,86 @@ START_TEST(default_config_test)
 END_TEST
 
 
+START_TEST(sequential_print_check)
```

Comment:
There is a rule for unit testing of one assert per test. That is a bit officious but we should aim for close to this. If you need more it might be a sign that you need more in setup and teardown or we need to add parametrized test capability.